metadata-ingestion/setup.py

```diff
@@ -133,7 +133,10 @@ def get_long_description():
     # PyAthena is pinned with exact version because we use private method in PyAthena
     "athena": sql_common | {"PyAthena[SQLAlchemy]==2.4.1"},
     "azure-ad": set(),
-    "bigquery": sql_common | bigquery_common | {"pybigquery >= 0.6.0"},
+    # pybigquery is still needed for great_expectations < 0.14.7
+    # https://github.com/great-expectations/great_expectations/blob/develop/docs/changelog.md#0147
```

Comment:
can be remove pybigquery completly and bump great-expectation? 
```
"bigquery": sql_common | bigquery_common | {"sqlalchemy-bigquery >= 1.4.1"},
```
and 
```
"great-expectations>=0.14.11",
```
For me sqlalchemy picked up pybiquery instead of the new driver if it existed and only the latest great expectation works with the new Bigqueyr driver.
I tested that setup and it seems like Datahub profiling works with the latest great expectation if we have only the new bigquery driver. 
I also tested profiling with this setup on Snowflake, Redshift as well.